python/paddle/tensor/random.py

```diff
@@ -370,6 +370,40 @@ def gaussian(shape, mean=0.0, std=1.0, seed=0, dtype=None, name=None):
         return out
 
 
+@dygraph_only
+def gaussian_(x, mean=0.0, std=1.0, seed=0, name=None):
+    """
+    This is the inplace version of OP ``gaussian``, which returns a Tensor filled
+    with random values sampled from a gaussian distribution. The output Tensor will
+    be inplaced with input ``x``. Please refer to :ref:`api_tensor_gaussian`.
+
+    Args:
+        x(Tensor): The input tensor to be filled with random values.
+        mean (float|int, optional): Mean of the output tensor, default is 0.0.
+        std (float|int, optional): Standard deviation of the output tensor, default
+            is 1.0.
+        seed (int, optional): Random seed of generator.
+        name(str, optional): The default value is None. Normally there is no
+            need for user to set this property. For more information, please
+            refer to :ref:`api_guide_Name`.
+    Returns:
+        Tensor: The input tensor x filled with random values sampled from a gaussian
+        distribution.
+    Examples:
+        .. code-block:: python
+
+            >>>import paddle
+            # example:
+            >>>x = paddle.ones(shape=[3, 4])
+            >>>x.gaussian_()
+            >>>print(x)
+            # [[ 0.84524226,  0.6921872,   0.56528175,  0.71690357], # random
+            #  [-0.34646994, -0.45116323, -0.09902662, -0.11397249], # random
+            #  [ 0.433519,    0.39483607, -0.8660099,   0.83664286]] # random
+    """
+    return _C_ops.gaussian_inplace_(x, float(mean), float(std), int(seed))
```

Comment:
If we want to support the mean and variance of complex numbers, there will be problems with this processing

Reply:
will be replaced by scalar when adding complex numbers